_pages/ato/ssp.md

```diff
@@ -8,14 +8,26 @@ As described in [the NIST guide](http://csrc.nist.gov/publications/nistpubs/800-
 
 The system security plan (SSP) is a long Google Doc. 18F is actively working on a tool to generate them called [Compliance Masonry](https://github.com/opencontrol/compliance-masonry), which uses a machine-readable format for which template is below. For now, the Google Docs are the canonical versions of the SSP.
 
+### Tips
+
+* Remember that the reviewer knows nothing about your system, and likely isn't even technical. The purpose of the SSP is to get the entire system and everything security-related around it into the brain of the reviewer.
```

Comment:
Would "likely isn't a developer" work, or does that still come across as disparaging?